### DISCUSSION TITLE
Discord server?

### DISCUSSION THREAD

**Discussion by arminherling** [kdl-org/kdl#395] [Q&A] [answered]
Hi, i recently started working on a C++ KDL parser based on the 2.0 Draft and was wondering if there is a discord server.
It might be fun to create one to make discussing details easier and talk about implementations.

**Comment by tabatkins** [comment]
We do not have a discord currently, no. Might be fun to set one up, tho.

**Reply by arminherling** [reply]
If @zkat doesnt want to set one up, maybe one of the other contributors could set one up.

**Reply by danini-the-panini** [reply]
Perhaps a matrix room would be more appropriate

**Reply by zkat** [reply]
My main concern with Matrix, as someone who has tried to establish project communities there before, is that... people just don't use it. It's not gonna be much of a community if people aren't willing to jump on Matrix and hang out there. I'm otherwise completely fine with it. Element kinda sucks, but Discord are kinda not the best either (but their tools and accessibility are WAY better)

**Comment by tabatkins** [comment, marked as answer]
I've gone ahead and set up a Discord server. Currently fairly bare-bones, we'll expand as we see fit. (Kat's got more experience with dev community management, might have more opinions on how to structure the rooms.)

https://discord.gg/mCMra2Z3
